packages/odyssey-react/CHANGELOG.md

```diff
@@ -3,6 +3,13 @@
 All notable changes to this project will be documented in this file.
 See [Conventional Commits](https://conventionalcommits.org) for commit guidelines.
 
+### [0.8.4](https://github.com/okta/odyssey/compare/v0.8.3...v0.8.4) (2021-11-18)
+
+### Features
+
+- expose hidden choices reference ([a64597a](https://github.com/okta/odyssey/commit/a64597a57f41bbbb1053d9e1a7254148d9318875))
+- support async select options ([0bc1986](https://github.com/okta/odyssey/commit/0bc1986e01d6f843cd589c56d4cdda27b629750f))
```

Comment:
looks like the changes here didn't take effect - if its problematic for whatever reason lets get the release out and open another PR in the `develop` branch to fix after we merge the release tag back in in phase 3 cleanup